docs/agents/magpie.rst

```diff
@@ -335,8 +337,5 @@ Supporting APIs
 .. autoclass:: agents.magpie.magpie_agent.FIRFilter
   :members:
 
-.. autoclass:: agents.magpie.magpie_agent.RollingAvg
-  :members:
-
 .. autoclass:: agents.magpie.magpie_agent.FocalplaneConfig
   :members:
```

Comment:
Can you add the other supporting classes here?

Reply:
Added!